.github/workflows/design-consistency-review.yml

```
@@ -0,0 +1,62 @@
+            このPRについて、コードと設計ドキュメント（docs/design/）に異なる点がないかをレビューしてください。
+
+            ## チェック観点
+
+            コードとドキュメントの間で以下のような相違点がないか確認してください：
+            - 構造体やメソッドの定義が一致しているか
+            - 設定項目やデフォルト値が一致しているか
+            - コマンドライン引数の仕様が一致しているか
+            - ビルトインコマンドの仕様と実装が一致しているか
+
+            ## レビュー方針
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)